### PR TITLE
Fix LIKE scan

### PR DESCRIPTION
**Pull request by mrzzzrm**
`LIKE` scans with parentheses in the pattern were broken
`LIKE` scans with some special regex chars in the pattern where broken (got escaped twice)

### CODE REVIEW

File: src/test/tables/int_string_like.tbl [hunk 1]
```diff
@@ -5,4 +5,4 @@ int|string
 12345|Reeperbahn
 123456|Dampfschifffahrtsgesellschaftskapitän
 1234567|Dampfschifffahrtsgesellschaftskapitänsdampf
-1234567|1234
+1234567|1234
```
Comment (mrks) [inline src/test/tables/int_string_like.tbl]: ?

Comment (mrzzzrm) [reply src/test/tables/int_string_like.tbl]: resetted to master

**Comment by mrzzzrm** [comment]
The JOB uses LIKE like quite heavily...
image

**Comment by mrks** [comment]
Fancy visualization!

**Comment by mrzzzrm** [comment]
Our statistic component thinks that there are 640'000 Robert Dawneys in this world....

**Pull request merged**